hudi-common/src/test/java/org/apache/hudi/common/util/collection/TestRocksDBManager.java

```diff
@@ -99,25 +104,119 @@ public void testRocksDBManager() {
         List<Pair<String, Payload>> gotPayloads =
             dbManager.<Payload>prefixSearch(family, prefix).collect(Collectors.toList());
         Integer expCount = countsMap.get(family).get(prefix);
+        System.out.printf("%s,%s: %d, %d\n", prefix, family, expCount == null ? 0L : expCount.longValue(), gotPayloads.size());
```

Comment:
can we make this log ?

Reply:
this was just a debug statement. so removed it